### PR TITLE
fixed problems

### PR DESCRIPTION
**Pull request by ItsMatDev**

### CODE REVIEW

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  height: 100vh;
+  background-size: cover;
+  width: 20vw;
+}
+button {
+  background-color: $primaryColor;
+  font-size: 1.3rem;
+  font-weight: bold;
+  width: 10vw;
+  height: 3vw;
```
Comment (pagouillart) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: vw ?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  border: 2px solid $secondaryColor;
+  text-align: center;
+  position: relative;
+}
+.card {
+  height: 100vh;
+  background-size: cover;
+  width: 20vw;
+}
+button {
```
Comment (pagouillart) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: class?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  background-color: $primaryColor;
+  font-size: 1.3rem;
+  font-weight: bold;
+  width: 10vw;
+  height: 3vw;
+  border-radius: 10% 30% 50% 70%;
+  a {
```
Comment (pagouillart) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: class?

Comment (pagouillart) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: > class?
décalé

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        {universe.map((monde) => (
+          <li key={monde.name}>
+            <article
+              className="card"
+              style={{
+                backgroundImage: `url(${monde.imgSrc})`,
+              }}
+            >
+              <h2 className="titleEra">{monde.name}</h2>
+              <button type="button">Jouer</button>
```
Comment (pagouillart) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: class?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import "./ChooseLevelPage.scss";
+import jurassique from "../../assets/pictures/Jurassique.png";
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: No French in the code

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+function ChooseLevelPage() {
+  const universe = [
+    { name: "CRÉTACÉ", imgSrc: jurassique, page: "./levelPage.jsx" },
+    { name: "ROMAIN", imgSrc: romain },
+    { name: "EGYPTE", imgSrc: egypte },
+    { name: "MOYEN-AGE", imgSrc: moyenAge },
+    { name: "MODERNE", imgSrc: moderne },
+  ];
+  return (
+    <div>
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: Why a div tag ?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      <ul>
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: Why an ul ?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+        {universe.map((monde) => (
+          <li key={monde.name}>
+            <article
+              className="card"
+              style={{
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: Never an inline style, it's not the first time I've said that.

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 9]
```diff
@@ -0,0 +1,43 @@
+@import "../../variables.scss";
+ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: flex-direction is row by default

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+  width: 100vh;
+}
+li {
+  color: $primaryColor;
+  list-style: none;
+  border: 2px solid $secondaryColor;
+  text-align: center;
+  position: relative;
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: Why relative position ?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 1 above)
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: Not vw for height

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+  width: 10vw;
+  height: 3vw;
+  border-radius: 10% 30% 50% 70%;
+  a {
+    text-decoration: none;
+    color: $secondaryColor;
+  }
+}
+h2 {
+  margin-bottom: 35rem;
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: Why such a low margin? Is it responsive?

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 1 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 2 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 3 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx (hunk 4 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx (hunk 5 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx (hunk 7 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx (hunk 8 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 9 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 10 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 1 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 11 above)
Comment (ItsMatDev) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: done

**Review by pagouillart — APPROVED** [review]

**Review by pagouillart — APPROVED** [review]

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx [hunk 12]
```diff
@@ -1,36 +1,29 @@
...
-            >
-              <h2 className="titleEra">{monde.name}</h2>
-              <button type="button">Jouer</button>
-            </article>
-          </li>
-        ))}
-      </ul>
-    </div>
+    <main className="cardContainer">
+      {universe.map((monde) => (
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: No Franglais

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss [hunk 13]
```diff
@@ -1,43 +1,49 @@
...
-  background-color: $primaryColor;
-  font-size: 1.3rem;
-  font-weight: bold;
-  width: 10vw;
-  height: 3vw;
-  border-radius: 10% 30% 50% 70%;
-  a {
-    text-decoration: none;
-    color: $secondaryColor;
+  &CRÉTACÉ {
```
Comment (Yoann-Fortin) [inline frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: No Franglais

**Review by Yoann-Fortin — APPROVED** [review]

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx (hunk 12 above)
Comment (Bravax) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.jsx]: monde modified to world

File: frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss (hunk 13 above)
Comment (Bravax) [reply frontend/src/components/ChooseLevelPage/ChooseLevelPage.scss]: We named our className according to the BEM method, which induces the written words based on the display of the table name " universe = [    { name: "CRÉTACÉ" },    { name: "ROMAIN" },    { name: "EGYPTE" },    { name: "MOYEN-AGE" },    { name: "MODERNE" }  ];     Is this valid?

**Pull request merged**